pkg/bus/chat.go

```diff
@@ -36,6 +36,16 @@ func (b *Bus) chat(ctx context.Context, userID string, evt *pb.ChatMessage) erro
 		return err
 	}
 
+	// Regulate chat only if the user is not privileged and the
+	// chat is not a game chat
+	regulateChat := "regulated"
```

Comment:
Can this (and the corresponding variable in the lua script) be a boolean?

Reply:
i first tried boolean, but i ran into problems i didn't understand

Reply:
hmm, maybe @domino14 can have a look, but not a huge deal either way.

Reply:
these could still be boolean, but the receiving function `AddChat` would take care of the actual implementation details (turn it into "regulated")